### PR TITLE
Use versioned gas configuration to modify or add gas entries on-chain

### PR DESCRIPTION
**Pull request by steelgeek091**

### CODE REVIEW

**Review by jolestar — APPROVED** [review]
Add a CI task to generate genesis.

**Comment by steelgeek091** [comment]
> Add a CI task to generate genesis.

Fixed

File: crates/rooch-genesis/src/lib.rs [hunk 1]
```diff
@@ -300,8 +300,10 @@ impl RoochGenesis {
                 || network.chain_id == BuiltinChainID::Local.chain_id()
             {
                 FrameworksGasParameters::latest()
-            } else {
+            } else if network.chain_id == BuiltinChainID::Main.chain_id() {
                 FrameworksGasParameters::initial()
+            } else {
+                FrameworksGasParameters::v1()
             }
```
Comment (jolestar) [inline crates/rooch-genesis/src/lib.rs:307]: 这里可以换个写法

if main initial
else if test v1
else latest

Comment (steelgeek091) [reply crates/rooch-genesis/src/lib.rs]: Fixed

**Pull request merged**